### DISCUSSION TITLE
Use of bind raises no-unsafe-argument error

### DISCUSSION THREAD

**Discussion by ghost** [eslint/eslint#16460] [General]
Having converted our Angular 14 project to use ESLint we are now getting errors relating to the usage of the bind method for example: -

````
this.formGroup
    .get('findingText')
    .setValidators(
        CustomValidators.isRequiredBecauseNonConformant(this.viewModel).bind(this)
    );
````

this raises following error, if i remove the `.bind(this)` the error goes away!

````
[{
	"resource": "/c:/SourceCode/Manatee/MobileApp3/cchub/src/app/audit/audit-group/audit-response/components/audit-question/audit-question.component.ts",
	"owner": "eslint",
	"code": {
		"value": "@typescript-eslint/no-unsafe-argument",
		"target": {
			"$mid": 1,
			"external": "https://typescript-eslint.io/rules/no-unsafe-argument",
			"path": "/rules/no-unsafe-argument",
			"scheme": "https",
			"authority": "typescript-eslint.io"
		}
	},
	"severity": 8,
	"message": "Unsafe argument of type `any` assigned to a parameter of type `ValidatorFn | ValidatorFn[]`.",
	"source": "eslint",
	"startLineNumber": 209,
	"startColumn": 21,
	"endLineNumber": 211,
	"endColumn": 33
}]
````

I don't want to disable this rule as it is very useful 99% of the time and there are no config options available for it, any ideas?

**Comment by mdjermanovic** [comment]
> "value": "@typescript-eslint/no-unsafe-argument",

This is a rule from `@typescript-eslint/eslint-plugin` so you will probably get a better answer in the plugin's repository:

https://github.com/typescript-eslint/typescript-eslint

This issue seems related:

https://github.com/typescript-eslint/typescript-eslint/issues/4001
